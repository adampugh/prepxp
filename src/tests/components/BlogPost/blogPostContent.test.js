import React from "react";
import { shallow } from "enzyme";

import BlogPostContent from "../../../components/BlogPost/blogPostContent";
import posts from "../../fixtures/blogPosts";


describe("<BlogPostContent />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<BlogPostContent post={posts[0]}/>);
        expect(wrapper).toMatchSnapshot();
    });
});