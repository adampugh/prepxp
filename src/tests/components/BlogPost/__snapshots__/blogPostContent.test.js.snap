// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlogPostContent /> should match snapshot 1`] = `
<div
  className="blogPost__content"
>
  <div
    className="container-s"
  >
    <div
      className="blogPost__content__panel"
    >
      <h1>
        5 Tips to get things done
      </h1>
      <h2
        className="blogPost__content__byline"
      >
        yesyesyeys
      </h2>
      <img
        alt="blog graphic"
        src="http://via.placeholder.com/300x300"
      />
      <div
        id="blogPost__content__text"
      />
    </div>
  </div>
</div>
`;
